run_raster_stats.py

```diff
@@ -110,19 +116,28 @@ def process_chunk(start, end, dataset, mode, df_iso3s, engine_url):
 
 if __name__ == "__main__":
     args = cli_args()
-    dataset = args.dataset
-    logger.info(f"Updating data for {dataset}...")
 
     engine_url = db_engine_url(args.mode)
     engine = create_engine(engine_url)
 
+    if args.update_metadata:
+        logger.info("Updating metadata in Postgres database...")
+        create_iso3_df(engine)
+        process_polygon_metadata(
+            engine,
+            args.mode,
+            upsampled_resolution=UPSAMPLED_RESOLUTION,
+            sel_iso3s=None,
+        )
+        sys.exit(0)
```

Comment:
Why this sys.exit here?

Reply:
Just because the program can terminate here after updating the metadata!